lib/gui/app/models/storage.js

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
This needs to be rebased on https://github.com/resin-io/etcher/pull/2276 and https://github.com/resin-io/etcher/pull/2311